### PR TITLE
Clean up docs

### PR DESCRIPTION
**Pull request by raphaellaude**
## Description

- Just cleaning up a bit before more people start to look at the repo!

## Reviewers

- Primary: @mariogiampieri 
- Secondary: @bailliekova 

## Checklist

- [x] Added/Updated related documentation (if applicable).
- [ ] Added/Updated related unit tests (if applicable).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,34 @@
...
-TK
+The redesign aims to principally to address three key pain points in the Districtr application’s performance and maintainability:
+1. Slow tile rendering
+1. Cumbersome use of tiles as global state for tile rendering and most metric calculation
+1. Complexity and poor interoperability in architecture without slow copies
 
-### Pipelines
+And two key feature additions
+1. Block “shattering”
+1. A headless CMS
```
Comment (mariogiampieri) [inline README.md:25]: should be "2" and the CMS is out of our scope right? if so we should indicate that it is for a future phase imo

**Review by mariogiampieri — CHANGES_REQUESTED** [review]
looks good minus the numbering issue in the base readme

File: README.md (hunk 1 above)
Comment (raphaellaude) [reply README.md:25]: Will add a note that CMS is out of scope for now. As for the numbering, MD is smart and will number lists automatically so you don't need to worry about always keeping the numbers ordered.

image

**Review by mariogiampieri — APPROVED** [review]
looks good!

**Pull request merged**